api/db/changelog/db.changelog-1.xml

```diff
@@ -60,7 +60,9 @@
       <column name="name" type="varchar(80)">
         <constraints nullable="false"/>
       </column>
-      <!-- The namespace and name of the workspace in Firecloud, e.g. myproject/workspace_name -->
+      <column name="project_name" type="varchar(255)">
```

Comment:
I think I'm agreed here. I'm happy to use firecloud_name as our id, but I think I prefer workspace_namespace to project_name